src/components/api-request.js

```diff
@@ -692,7 +692,7 @@ export default class ApiRequest extends LitElement {
 
   onClearRequestData(e) {
     const requestPanelEl = e.target.closest('.request-panel');
-    const requestPanelInputEls = [...requestPanelEl.querySelectorAll('input, tag-input, textarea:not(.is-hidden)')];
+    const requestPanelInputEls = [...requestPanelEl.querySelectorAll('input, select, tag-input, textarea:not(.is-hidden)')];
```

Comment:
I actually think `select` needs special handling, for instance what if select doesn't have a `' '` option, for optional enums for instance, the values include `null` not `' '`. And for required enums `' '` isn't an option. Realistically we would need to select the default value in cases where the parameter is required and the nullish version that matches the property for types were it is required. I don't know if the properties already exist to figure that out from the html attributes....